OpenOversight/app/templates/403.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block title %}Forbidden{% endblock %}
+
+{% block content %}
+
+<div class="container theme-showcase" role="main">
+
+<h1>Forbidden</h1>
+<p>You do not have permissions to view this page. <a href="{{ url_for('main.get_tagger_gallery') }}">Return to homepage</a>.</p>
```

Comment:
why are these all linking to `get_tagger_gallery()`? that's a weird place to end up compared to `index()`, no?